hedera-node/src/test/java/com/hedera/services/txns/ethereum/EthTxDataTest.java

```diff
@@ -176,18 +180,66 @@ void whiteBoxDecodingErrors() {
 				oneByte, oneByte, oneByte, oneByte,
 				oneByte, oneByte, oneByte, oneByte,
 				oneByte);
+		var size_1 = List.of(
+				oneByte);
 
 		// legacy TX with too many RLP entries
 		assertNull(EthTxData.populateEthTxData(RLPEncoder.encodeAsList(size_13)));
+		// legacy TX with too few RLP entries
+		assertNull(EthTxData.populateEthTxData(RLPEncoder.encodeAsList(size_1)));
 		// type 2 TX with too many RLP entries
 		assertNull(EthTxData.populateEthTxData(RLPEncoder.encodeSequentially(new byte[] { 2 }, size_13)));
+		// type 2 TX with too few RLP entries
+		assertNull(EthTxData.populateEthTxData(RLPEncoder.encodeSequentially(new byte[] { 2 }, size_1)));
 		// Unsupported Transaciton Type
 		assertNull(EthTxData.populateEthTxData(RLPEncoder.encodeSequentially(new byte[] { 127 }, size_13)));
+		// Trimmed End Bytes
+		assertThrows(IllegalArgumentException.class, () ->
```

Comment:
There we will have sonar code smell: Only one method invocation is expected when testing runtime exceptions

Reply:
Sonar was right, the coding of the test was not correct.  We shouldn't re-encode the already encoded RLP.

This exposes a bug, we should be returning null not throwing exceptions.